yarn.lock

```diff
@@ -11769,7 +11769,7 @@ __metadata:
     yargs-parser: ^16.1.0
   bin:
     gluegun: bin/gluegun
-  checksum: 39134821c7233bae61fa0d85bcc5ab4ccd2cbe00173c53cc3cab74c4f29d54de939809832fc4e9bedb896a0795313d28f854bc7d5abe7b154bf7d77c319398bd
+  checksum: f7a573447561af36ebe5012a2fee68df90d8a6287ac1c8aaa6d7a95c82480181ca66ba7e3d5dbeaa672bab14fd736c85806b2049926a4100a14cec5d80f53413
```

Comment:
Let's keep yarn.lock as well. If it doesn't work properly on your end, you can just update it locally.